pkgs/top-level/all-packages.nix

```diff
@@ -10835,6 +10835,10 @@ with pkgs;
 
   nginxModules = callPackage ../servers/http/nginx/modules.nix { };
 
+  nginxShibboleth = callPackage ../servers/http/nginx/stable.nix {
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders nginxModules.shibboleth ];
+  };
```

Comment:
@Mic92 @copumpkin I would like to have dynamic module support as well, but the nginx generic.nix currently doesn't seem to have dynamic module compilation support:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/http/nginx/generic.nix#L48 which is not "--add-dynmaic-module".

I'd typically do an override, but this is for an elastic fleet of servers, and can't have to re-compile nginx on every server.